ClosedXML_Tests/Excel/Cells/XLCellTests.cs

```diff
@@ -369,6 +369,36 @@ public void ValueSetDateWithShortUserDateFormat()
             Assert.AreEqual(expected, actual);
         }
 
+        [Test]
+        public void SetStringCellValues()
```

Comment:
I would suggest adding a few more test cases:

* Specify an empty string
* Specify a string longer than 32767 characters
* Specify null
* Specify number as a string
* Specify DateTime less than `BaseDate`
* Specify DateTime, Decimal, Integer, TimeSpan, Boolean without converting to strings
* Specify `FormulaA1` and then set cell value and ensure that both `FormulaA1` and `FormulaR1C1` are empty
* Specify a string having line break included and ensure that `WrapText` is set to true
* Specify a string containing a single quote and number (e.g. `'10`) and make sure the `DataType` is Text.

Perhaps some of those cases are already covered with tests but they can help not to get into regression in future.